### PR TITLE
Et100

### PR DESCRIPTION
**Pull request by stacylozovan**

### CODE REVIEW

File: src/entity/entity/Entity.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package entity;
```
Comment (patricksanto) [inline src/entity/entity/Entity.java]: folder structure for entity class does not seem right .

File: src/entity/entity/Entity.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    public Entity() {
+        this.x = 0;
+        this.y = 0;
+        this.speed = 0;
+        this.direction = "down";
+        this.health = 100;
+    }
+
+
+    public void takeDamage(int damage) {
```
Comment (patricksanto) [inline src/entity/entity/Entity.java]: lets just change the code necessary for the PR. take damage is suposed to be implemented in the battle task.

File: src/entity/CharacterManager.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+public class CharacterManager {
```
Comment (patricksanto) [inline src/entity/CharacterManager.java:1]: Character manager is suposed to be responsible for the creation of the characters .

File: src/main/Main.java [hunk 4]
```diff
@@ -5,20 +5,26 @@
...
 
     @Override
     public void init() {
         player = new Player();
         player.setDefaultValues();
 
+        madara = new Madara();
+        madara.setDefaultValues();
+
+        characterManager = new CharacterManager(player, madara);
```
Comment (patricksanto) [inline src/main/Main.java]: better design solution could be a method inside Charachtermanager called .createCharachter() or .createBoss() , and pass all the necessaries informations .  and in the main just call another more global method e.g. charachterManager.generateAll() , which gonna call all methods and create all necessaries characters.

File: src/entity/Player.java [hunk 5]
```diff
@@ -90,7 +90,14 @@ public void draw() {
         }
         SaxionApp.drawImage(image, x, y, 64, 64);
     }
+    public void takeDamage(int damage) {
```
Comment (patricksanto) [inline src/entity/Player.java:93]: lets keep the PR clean with just the necessary code.

File: src/entity/CharacterManager.java [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+    private Player player;
+    private Madara madara;
+
+    public CharacterManager() {
```
Comment (patricksanto) [inline src/entity/CharacterManager.java:6]: this is great

**Review by patricksanto — APPROVED** [review]

**Review by NaufalS9 — APPROVED** [review]

**Pull request merged**